packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1279,6 +1287,10 @@ class PopupMenuButton<T> extends StatefulWidget {
   /// over the button that was used to create it.
   final PopupMenuPosition? position;
 
+  /// Determines the duration of time it takes for the popup menu transition to
```

Comment:
The conventional format for API doc is one sentence that summarizes the property and then a blank line and then additional details (also: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose).

In this case it would be helpful to explain how this duration applies to both opening and closing the menu. If we're leaving out a pair of Animation curves (forward and reverse) then we should consider including them as well. 

And finally, since this property might establish a pattern that could be followed by all of the other menu classes and menu button classes, it would be helpful to think about how best to encapsulate the duration[s] and curve[s] in one object; since they collectively modify how a simple animated transition will happen. Better than than adding other curves and durations ad-hoc.